src/components/settings/cameras/CameraConfigDialog.vue

```diff
@@ -85,7 +85,8 @@
             { text: $t('app.setting.camera_type_options.hlsstream'), value: 'hlsstream' },
             { text: $t('app.setting.camera_type_options.webrtc_camera_streamer'), value: 'webrtc-camerastreamer' },
             { text: $t('app.setting.camera_type_options.video'), value: 'ipstream' },
-            { text: $t('app.setting.camera_type_options.iframe'), value: 'iframe' }
+            { text: $t('app.setting.camera_type_options.iframe'), value: 'iframe' },
+            { text: $t('app.setting.camera_type_options.webrtc_gortc'), value: 'webrtc-gortc' }
```

Comment:
I know that Mainsail PR has not been merged yet, but we should attempt to use the same type here on both UIs.

```suggestion
            { text: $t('app.setting.camera_type_options.webrtc_gortc'), value: 'webrtc-go2rtc' }
```

Reply:
if I use the go2rtc, this doesn't map correctly. 

   ``` if (this.camera.service) {
      const componentName = `${this.$filters.startCase(this.camera.service).replace(' ', '')}Camera`

      if (componentName in CameraComponents) {
        return CameraComponents[componentName]
      }```

Reply:
![image](https://github.com/fluidd-core/fluidd/assets/152501/09b0728a-24d4-49f3-ab1a-0bf0bb555df9)

See here the startCase mangles up the name adding spaces. What do you suggest here?

Reply:
I can replace the `' ',''`  again after it converts it to startCase?